api/queuesvc/processors/processor.go

```diff
@@ -374,7 +373,7 @@ func resolveParentInfo(h *handler.H, sub *types.Submission) (*types.Submission,
 		return nil, eErr
 	}
 
-	token := &oauth2.Token{}
+	token := &types.OAuthToken{}
```

Comment:
I know there's no import conflict in this file, but the inconsistency of `t` and `types` between files might catch me off guard.

Reply:
there's two `types` packages in the system; in the case `t` is used both are in use.

Reply:
I understand your intention, I just mean I may find it strange if I'm flipping between files and have to remember that `types` refers to different packages depending on the file I'm in. Maybe it's ok, was a thought I had reading through.